optimade/filtertransformers/mongo.py

```diff
@@ -389,6 +393,43 @@ def replace_with_relationship(subdict, prop, expr):
             filter_, check_for_entry_type, replace_with_relationship
         )
 
+    def _apply_unknown_or_null_filter(self, filter_: dict) -> dict:
+        """ This method loops through the query and replaces the check for
+        KNOWN with a check for existence and a check for not null, and the
+        inverse for UNKNOWN.
+
+        """
+
+        def check_for_known_filter(prop, expr):
```

Comment:
```suggestion
        def check_for_known_filter(_, expr):
```
Why was this not found in the linting?

Reply:
#269 still holds, should really turn off silent fails for flake8...